methods/did:.md

```diff
@@ -0,0 +1,33 @@
+# did:*
+
+The "any" DID method encoding allows any DID to be represented as a multidid by essentially putting the entire DID ASCII string in the URL portion of the multidid byte string.
+
+## Method multicodec
+
+`0x00`
+
+## Format
+
+```
+<multidid-code><method-code><method-bytes><url-length><url-bytes>
+```
+
+Where:
+
+* `multidid-code` - a varint encoding of `0xd1d`
+* `method-code` - a varint encoding of `0x00`
```

Comment:
Why is this set to "identity multihash" do you mean `0x55` raw IPLD ?

Reply:
yeah, that's probably better.